src/IECoreScene/bindings/ShaderNetworkAlgoBinding.cpp

```diff
@@ -61,6 +61,11 @@ CompoundDataPtr expandSplineParametersWrapper( CompoundDataPtr parameters )
 	return boost::const_pointer_cast< CompoundData >( ShaderNetworkAlgo::expandSplineParameters( parameters ) );
 }
 
+IECore::InternedString componentConnectionAdapterLabelWrapper()
```

Comment:
Worth converting this to a regular Python string on return I think - InternedStrings are in a bit of an uncanny valley when it comes to Python.